src/packages/ganache/api-extractor.json

```diff
@@ -347,9 +348,6 @@
         // api-extractor _really_ wants to complain about this one, so much so
         // that setting the `"default"` to "none" (above) has no affect!
         "logLevel": "none"
-      },
-      "ae-wrong-input-file-type": {
-        "logLevel": "none"
```

Comment:
We previously added this flag because we were updating the api-extractor version. Now we aren't, so we can remove